tests/golos.emit_tests.cpp

```diff
@@ -18,66 +18,39 @@ class golos_emit_tester : public golos_tester {
     golos_emit_api emit;
     cyber_token_api token;
 
+    const account_name issuer = cfg::issuer_name;
+
 public:
     golos_emit_tester()
         : golos_tester(cfg::emission_name)
-        , emit({this, _code})
+        , emit({this, cfg::emission_name})
```

Comment:
the initial idea was to set contract name in `golos_tester()`, and then use `_code` as account_name of the main contract (for which test for). here it fits right for me, propose revert